index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/171fcd9d48.js" crossorigin="anonymous"></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Geolex Cyber Space</title>
+  </head>
+
+  <body>
+    <header>
+      <!-- top social icons -->
+      <div class="social-section">
+        <div class="top-nav">
+          <a href="#" class="facebook">facebook</a>
+          <a href="#" class="twitter">twitter</a>
+          <a href="#" class="english">English</a>
+          <a href="#" class="mypage">My page</a>
+          <a href="#" class="logout">logout</a>
+        </div>
+      </div>
+
+      
+      <nav class="nav">
+        <!-- logo -->
+        <h3 class="logo">
+          <a class="logo" href="index.html">Geolex</a>
+        </h3>
+
+        <div class="nav-menu">
+          <ul class="menu-list">
+            <li class="list"><a href="index.html">Home</a></li>
+            <li class="list"><a href="about.html">About</a></li>
+            <li class="list"><a href="#services"></a>Services</li>
+            <li class="list"><a href="contact.html"></a>Contact</li>
+            <li class="list"><a href="#"></a>Extras</li>
+            <li class="list"><a href="#"></a>SignUp</li>
+          </ul>
+
+        </div>
+
+        <!-- Hamburger -->
+        
+      </nav>
+      <div class="hamburger">
+        <div class="bar ham1"></div>
+        <div class="bar ham2"></div>
+        <div class="bar ham3"></div>
+      </div>
```

Comment:
As requested by the previous reviewer, the hamburger menu is yet to work as expected. It does not work on the about page and an attempt to navigate to the about page using the menu only makes the menu disappear without navigating.

How it works currently:

https://user-images.githubusercontent.com/20554583/190837610-81072c2e-ddfa-4934-a9e1-7bd8b2ad72fa.mp4



Kindly fix that so that the menu options will be properly accessible on mobile devices.

Reply:
I have resolved the error. thank you for your assistance